include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+    //VkBool32                             shaderStorageImageArrayNonUniformIndexingNative;
+    //VkBool32                             shaderInputAttachmentArrayNonUniformIndexingNative;
+    //VkBool32                             robustBufferAccessUpdateAfterBind;
+    //VkBool32                             quadDivergentImplicitLod;
+    //uint32_t                             maxPerStageDescriptorUpdateAfterBindSamplers;
+    //uint32_t                             maxPerStageDescriptorUpdateAfterBindUniformBuffers;
+    //uint32_t                             maxPerStageDescriptorUpdateAfterBindStorageBuffers;
+    //uint32_t                             maxPerStageDescriptorUpdateAfterBindSampledImages;
+    //uint32_t                             maxPerStageDescriptorUpdateAfterBindStorageImages;
+    //uint32_t                             maxPerStageDescriptorUpdateAfterBindInputAttachments;
+    //uint32_t                             maxPerStageUpdateAfterBindResources;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindSamplers;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindUniformBuffers;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindUniformBuffersDynamic;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindStorageBuffers;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindStorageBuffersDynamic;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindSampledImages;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindStorageImages;
+    //uint32_t                             maxDescriptorSetUpdateAfterBindInputAttachments;
+    //VkResolveModeFlags                   supportedDepthResolveModes;
+    //VkResolveModeFlags                   supportedStencilResolveModes;
+    //VkBool32                             independentResolveNone;
+    //VkBool32                             independentResolve;
+    //VkBool32                             filterMinmaxSingleComponentFormats;
+    //VkBool32                             filterMinmaxImageComponentMapping;
+    //uint64_t                             maxTimelineSemaphoreValueDifference;
+    //VkSampleCountFlags                   framebufferIntegerColorSampleCounts;
+            
+    /* Vulkan 1.3 Core  */
+    //uint32_t              minSubgroupSize;
+    //uint32_t              maxSubgroupSize;
+    //uint32_t              maxComputeWorkgroupSubgroups;
+    //VkShaderStageFlags    requiredSubgroupSizeStages;
+    //uint32_t              maxInlineUniformBlockSize;
+    //uint32_t              maxPerStageDescriptorInlineUniformBlocks;
+    //uint32_t              maxPerStageDescriptorUpdateAfterBindInlineUniformBlocks;
+    //uint32_t              maxDescriptorSetInlineUniformBlocks;
+    //uint32_t              maxDescriptorSetUpdateAfterBindInlineUniformBlocks;
+    //uint32_t              maxInlineUniformTotalSize;
```

Comment:
We don't expose inline uniform blocks in out descriptor sets/pools

Reply:
yeah, lets not expose now for now

Reply:
but mark as a future TODO